tests/acceptance/features/coreApiVersions/fileVersions.feature

```diff
@@ -48,14 +45,12 @@ Feature: dav-versions
       | spaces           |
 
 
-  Scenario Outline: upload a file twice and versions are available using various chunking methods (except new chunking)
+  Scenario Outline: upload a file twice and versions are available using various chunking methods
     Given using <dav-path-version> DAV path
-    When user "Alice" uploads file "filesForUpload/davtest.txt" to filenames based on "/davtest.txt" with all mechanisms except new chunking using the WebDAV API
-    And user "Alice" uploads file "filesForUpload/davtest.txt" to filenames based on "/davtest.txt" with all mechanisms except new chunking using the WebDAV API
+    When user "Alice" uploads file "filesForUpload/davtest.txt" to "davtest.txt" with and without chunking using the WebDAV API
```

Comment:
why uploading with and without at once?
```suggestion
    When user "Alice" uploads file "filesForUpload/davtest.txt" to "davtest.txt" with and without chunking using the WebDAV API
```

Reply:
>Scenario Outline: upload a file twice and versions are available using various chunking methods

Reply:
actual scenario: https://github.com/owncloud/ocis/pull/10624#discussion_r1881454112